arduino/smartcar/smartcar.ino

```diff
@@ -26,29 +38,34 @@ ArduinoRuntime arduinoRuntime;
 BrushedMotor leftMotor(arduinoRuntime, smartcarlib::pins::v2::leftMotorPins);
 BrushedMotor rightMotor(arduinoRuntime, smartcarlib::pins::v2::rightMotorPins);
 DifferentialControl control(leftMotor, rightMotor);
+ 
+GY50 gyroscope(arduinoRuntime, 37);
+const auto pulsesPerMeter = 600;
+DirectionlessOdometer leftOdometer(
+    arduinoRuntime, smartcarlib::pins::v2::leftOdometerPin, []() { leftOdometer.update(); },
+    pulsesPerMeter);
+DirectionlessOdometer rightOdometer(
+    arduinoRuntime, smartcarlib::pins::v2::rightOdometerPin, []() { rightOdometer.update();
+}, pulsesPerMeter);
+ 
+SmartCar car(arduinoRuntime, control, gyroscope, leftOdometer, rightOdometer);
 
-SimpleCar car(control);
-
-//infrared sensor//
-const int frontIRPin = 0;
-const int leftIRPin = 1;
-const int rightIRPin = 2;
-const int backIRPin = 3;
-GP2Y0A02 frontIR(arduinoRuntime, frontIRPin); 
-GP2Y0A02 leftIR(arduinoRuntime, leftIRPin); 
-GP2Y0A02 rightIR(arduinoRuntime, rightIRPin); 
-GP2Y0A21 backIR(arduinoRuntime, backIRPin); 
-    //measure infrared between 0 and 40
-
+// SimpleCar car(control);
```

Comment:
If you don't need this anymore then please remove it instead of commenting it out

Reply:
it was delete